src/services/api/media.js

```diff
@@ -51,9 +49,13 @@ const media = {
   updateAudio: async ({ id, sitename, data }) =>
     apiBase().put(`${SITES}/${sitename}/${AUDIO}/${id}`, { json: data }).json(),
   updateImage: async ({ id, sitename, data }) =>
-    apiBase().put(`${SITES}/${sitename}/${IMAGE}/${id}`, { json: data }).json(),
+    apiBase()
+      .put(`${SITES}/${sitename}/${IMAGE_PATH}/${id}`, { json: data })
```

Comment:
Great catch! Please switch over the uses of AUDIO in this file to AUDIO_PATH while you're at it. Thanks 👍